vald/config/config.go

```diff
@@ -46,8 +45,8 @@ type BroadcastConfig struct {
 // DefaultBroadcastConfig returns a configurations populated with default values
 func DefaultBroadcastConfig() BroadcastConfig {
 	return BroadcastConfig{
-		MaxRetries:          10,
+		MaxRetries:          3,
 		MinSleepBeforeRetry: 5 * time.Second,
-		MaxTimeout:          config.DefaultRPCConfig().TimeoutBroadcastTxCommit,
+		MaxTimeout:          15 * time.Second,
```

Comment:
why not change the default, if the timeout is not enough in general?

Reply:
We have bumped it in the config.toml we provide for folks. But the default value we get here is from the tendermint default config, can't change that.